### ISSUE TITLE
fix variable name generation

### ISSUE THREAD

**Issue by GitMensch** [vinshuka/Cobol-Pretty-Headers#5]
The README.md actually shows the issue, because of the generation of a variable with the name `Cobol!-HDR`. You may want to apply a regex or otherwise filter out/replace any "content" when building the variable name. Valid in all compilers is `[a-ZA-Z][a-ZA-Z0-9_-]*[a-ZA-Z0-9]`.

**Comment by vinshuka** [comment]
Just to clarify the issue is the inclusion of the '!' character in the variable name?

**Comment by GitMensch** [comment]
Yes, the same would happen with a lot of other "special" chars or with a leading non-alphabetic (a-z).
